dashboard/lib/projects_list.rb

```diff
@@ -116,7 +117,7 @@ def fetch_published_project_types(project_types, limit:, published_before: nil)
             exclude(published_at: nil).
             order(Sequel.desc(:published_at)).
             limit(limit).
-            map {|project_and_user| get_published_project_and_user_data project_and_user}
+            map {|project_and_user| get_published_project_and_user_data project_and_user}.compact
```

Comment:
Can we add a test for this? I'd like to verify that for a set of projects the sharing disabled ones are removed and the rest remain. I'd be ok with just 2 projects: stub the `PEGASUS_DB` call to return fake data with one sharing disabled, and the other enabled, then assert on the enabled one remains in the result.